README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
+
+Welcome to NI's internal and external JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started) and [TSLint](https://palantir.github.io/tslint/).
+
+## Installation
+
+```
+npm install --save-dev eslint eslint-plugin-import @ni/eslint-config
+```
+
+## Usage
+
+Extend @ni in the ESLint configuration.
+
+```json
+{
+    "extends": "@ni"
+}
+```
```

Comment:
We might also consider some tips about how to go about adopting our rules. Some combination of 
1. describing the commands to run the linter and the fixer
1. describe the workflow of disabling many rules and then enabling rules one-by-one
1. linking to eslint docs about how to configure rules

This could be something we add once we gain more experience in adopting the ruleset though.

Reply:
Sure! 
1. Install
2. Configure
3. Disable
...
Profit!

Naw, seriously all good ideas.

Reply:
This can wait until we've built up some best practices